LDAR_Sim/model_code/generic_functions.py

```diff
@@ -283,3 +283,44 @@ def quick_cal_daylight(date,lat,lon):
     sunset = ss
 
     return (sunrise,sunset)
+
+
+def find_homebase(x1,y1,HX,HY): 
+    XY = list(zip(HX,HY))
```

Comment:
Possibly a docstring here to help understand what these arguments are and what units they are in?

Reply:
@AkaMrmuffin